test/examples/test-zeus.good.ts

```diff
@@ -91,17 +91,17 @@ let tm = mutation(m => [
         Defense: 2,
         name: 'Hi',
         description: 'Lo',
-        skills: [$('skill')] as const,
+        skills: $('skills'),
       },
     },
     c => [c.Attack, c.Defense, c.Children]
   ),
 ])
 
 let tmString = `
-mutation ($skill: [SpecialSkills!]) {
```

Comment:
huh looks like there is an actual bug if the variable is embeded in an array. Will need to fix this afterwards.